components/jira/jira.app.mjs

```diff
@@ -1,11 +1,257 @@
+    },
+    async _makeRequest({
+      $,
+      path,
+      headers,
+      ...otherConfig
+    } = {}) {
+      const config = {
+        url: await this._getUrl($, path),
+        headers: this._getHeaders(headers),
+        ...otherConfig,
+      };
+      return axios($ ?? this, config);
+    },
+    async assignIssue({
+      issueIdOrKey,
+      ...args
+    } = {}) {
+      return await this._makeRequest({
+        method: "PUT",
+        path: `/issue/${issueIdOrKey}/assignee`,
+        ...args,
+      });
+    },
+    async addWatcher({
+      issueIdOrKey,
+      accountId,
+      ...args
+    } = {}) {
+      return await this._makeRequest({
+        method: "POST",
+        path: `/issue/${issueIdOrKey}/watchers`,
+        data: `"${accountId}"`,
+        ...args,
+      });
+    },
+    async addAttachmentToIssue({
+      issueIdOrKey,
+      ...args
+    } = {}) {
+      return await this._makeRequest({
+        method: "POST",
+        path: `/issue/${issueIdOrKey}/attachments`,
+        ...args,
+      });
+    },
+    async addCommentToIssue({
+      issueIdOrKey,
+      ...args
+    } = {}) {
+      return await this._makeRequest({
+        method: "POST",
+        path: `/issue/${issueIdOrKey}/comment`,
+        ...args,
+      });
+    },
+    async createIssue({ ...args } = {}) {
+      return await this._makeRequest({
+        method: "POST",
+        path: "/issue",
+        ...args,
+      });
+    },
+    async createVersion({ ...args } = {}) {
+      return await this._makeRequest({
+        method: "POST",
+        path: "/version",
+        ...args,
+      });
+    },
+    async deleteProject({
+      projectIdOrKey,
+      ...args
+    } = {}) {
+      return await this._makeRequest({
+        method: "DELETE",
+        path: `/project/${projectIdOrKey}`,
+        ...args,
+      });
+    },
+    async getAllProjects({ ...args } = {}) {
+      return await this._makeRequest({
+        method: "GET",
+        path: "/project",
+        ...args,
+      });
+    },
+    async getIssue({
+      issueIdOrKey,
+      ...args
+    } = {}) {
+      return await this._makeRequest({
+        method: "GET",
+        path: `/issue/${issueIdOrKey}`,
+        ...args,
+      });
+    },
+    async getTask({
+      taskId,
+      ...args
+    } = {}) {
+      return await this._makeRequest({
+        method: "GET",
+        path: `/task/${taskId}`,
+        ...args,
+      });
+    },
+    async getTransitions({
+      issueIdOrKey,
+      ...args
+    } = {}) {
+      return await this._makeRequest({
+        method: "GET",
+        path: `/issue/${issueIdOrKey}/transitions`,
+        ...args,
+      });
+    },
+    async getUser({
+      ...args
+    } = {}) {
+      return await this._makeRequest({
+        method: "GET",
+        path: "/user",
+        ...args,
+      });
+    },
+    async listIssueComments({
+      issueIdOrKey,
+      ...args
+    } = {}) {
```

Comment:
`listIssueComments` should handle pagination in case there are multiple pages of comments.

Reply:
Doing in this way, thanks!